opt-tools/docker/docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+version: '3'
+
+services:
+
+  postgres_db:
+    image: postgres
+    restart: always
+
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=school
+      - POSTGRES_DB=school
+
+    volumes:
+      - ../../data/postgres:/var/lib/postgresql/data
+      - ../../data/dump:/db-dump
+
+    ports:
+      - "5432:5432"
+
+    networks:
+      - docker-practice-net
+
+
+  redis:
+    image: "redis:alpine"
+    command: redis-server
+
+    ports:
+      - "6379:6379"
+
+    environment:
+      - REDIS_REPLICATION_MODE=master
+
+    networks:
+      - docker-practice-net
+
+
+  spring_app:
+
+    build:
+      context: ./images/app
+      dockerfile: Dockerfile
+
+    ports:
+      - "8888:8888"
+
+    network_mode: "host"
```

Comment:
Is required to use host network mode? does default mode not fit for our purposes?

Reply:
Okay, I'll remove this parameter